src/test/features/claimant-response/helpers/statesPaidHelper.ts

```diff
@@ -0,0 +1,146 @@
+import { expect } from 'chai'
+import { Claim } from 'claims/models/claim'
+import { sampleClaimObj } from '../../../http-mocks/claim-store'
```

Comment:
try to avoid relative path imports; I think this is `test/http-mocks/claim-store`